.gitignore

```diff
@@ -42,3 +42,6 @@ jspm_packages
 
 # Optional npm cache directory
 .npm
+
+#those darn DS_Store files
+**/.DS_Store
```

Comment:
~~Could've sworn I'd added those already!~~ They must be in my global gitignore.